webpack.common.ts

```diff
@@ -46,6 +52,17 @@ module.exports = {
   },
   module: {
     rules: [
+      {
+        test: /\.m?js$/,
+        include: MONACO_DIR,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env'],
+            plugins: ['@babel/plugin-transform-runtime'],
```

Comment:
Webpack plugin now uses ESM loading of the monaco-editor. This requires separate loading rules if encountered errors (which we did).

Reply:
Per discussion with @drdelambre , at the end of the final debugging (tests passing) I tested that these webpack updates are required. They are.

 Error without this rule:
```
ERROR in ./node_modules/monaco-editor/esm/vs/language/css/monaco.contribution.js 29:15
Module parse failed: Unexpected token (29:15)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| // src/language/css/monaco.contribution.ts
| var LanguageServiceDefaultsImpl = class {
>   _onDidChange = new monaco_editor_core_exports.Emitter();
|   _options;
|   _modeConfiguration;
 @ ./node_modules/monaco-editor/esm/vs/editor/editor.main.js 2:0-45
 @ include-loader!./node_modules/monaco-editor/esm/vs/editor/editor.main.js
 @ dll vendor
```

Error without the plugin '@babel/plugin-transform-runtime':
<img width="472" alt="Screen Shot 2022-04-06 at 5 05 50 PM" src="https://user-images.githubusercontent.com/10232835/162094324-58c654bf-91bb-4f4a-8134-ba6538b85156.png">

Reply:
k one more time.  now that we have removed the random worker that was being loaded and the only assets going through have  been verified as being compiled, meaning there's no need  for the runtime as it's baked in.... is this still needed?

Reply:
yes. Same error.

Still required for loading ESM modules for the bumped monaco-editor.